data/rufilboy.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Ilyas Rufai",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "*DevOps Engineer || Content Creator || Technical Writer*",
+    "avatar": "https://github.com/rufilboy.png",
```

Comment:
We no longer need the `avatar` key as of our latest update, it will auto-pick your GitHub profile. So Please remove this avatar field from your file. 

[Updated docs](https://linkfree.eddiehub.io/docs/quickstart)

```suggestion
```